### PR TITLE
remove findDOMNode

### PR DESCRIPTION
**Pull request by emma-floored**
@hwigmore @hamham91 this started breaking everything for some reason

### CODE REVIEW

File: react/lists/SortableList.jsx [hunk 1]
```diff
@@ -36,10 +36,11 @@ class SortableList extends React.Component {
   }
 
   componentDidMount() {
-    this.listDOM = ReactDOM.findDOMNode(this);
-    this.listDOM.addEventListener('dragstart', this.onDragStart);
-    this.listDOM.addEventListener('dragover', this.onDragOver);
-    this.listDOM.addEventListener('dragend', this.onDragEnd);
+    if (this.listDOM) {
```
Comment (hwigmore) [inline react/lists/SortableList.jsx]: would this ever not exist and if so when / how would these event listeners get added?

**Comment by hwigmore** [comment]
i had a question but overall makes sense and 👍

File: react/lists/SortableList.jsx (hunk 1 above)
Comment (emma-floored) [reply react/lists/SortableList.jsx]: I don't think it should ever not exist but I also didn't think the bug we were having in datavis should ever have happened... so I figured maybe at least in this case it wouldn't crash even if the list doesn't work

Comment (hwigmore) [reply react/lists/SortableList.jsx]: ok!

**Pull request merged**